### PR TITLE
docs(getting-started): add views page

### PR DESCRIPTION
**Pull request by clemcode**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
This PR explains how Views work in Nuxt:
- App.vue
- Page
- Layout
- Components

### CODE REVIEW

**Review by alexchopin — APPROVED** [review]
Well explained and good for me! Good work 🙌🏻

**Comment by pi0** [comment]
Made some tweaks! Nice work! Waiting for @danielroe to have second eyes on docs.

**Review by pi0 — APPROVED** [review]

File: docs/content/1.getting-started/3.views.md [hunk 1]
```diff
@@ -1,13 +1,130 @@
 # Views
 
-::NeedContribution
+Nuxt provides several component layers to implement the user interface of your application.
+
+## `app.vue`
+
+![The `app.vue` file is the entry point of your application](/img/getting-started/views/app.svg)
+
+By default, Nuxt will treat this file as the **entrypoint** and render it's content for every route of the application.
```
Comment (danielroe) [inline docs/content/1.getting-started/3.views.md:9]:
```suggestion
By default, Nuxt will treat this file as the **entrypoint** and render its content for every route of the application.
```

File: docs/content/1.getting-started/3.views.md [hunk 2]
```diff
@@ -1,13 +1,130 @@
...
+```vue [app.vue]
+<template>
+  <div>
+   <h1>Welcome to the homepage</h1>
+  </div>
+</template>
+```
+
+::alert
+If you are familiar with Vue, you might wonder where is the `main.js` file that creates a Vue app. Nuxt does this behind the scene.
```
Comment (danielroe) [inline docs/content/1.getting-started/3.views.md:20]:
```suggestion
If you are familiar with Vue, you might wonder where `main.js` is (the file is that normally creates a Vue app). Nuxt does this behind the scene.
```

File: docs/content/1.getting-started/3.views.md [hunk 3]
```diff
@@ -1,13 +1,130 @@
...
+::
+
+## Components
+
+![Components are reusable pieces of UI](/img/getting-started/views/components.svg)
+
+Most of the components are reusable pieces of the user interface, like buttons and menus. In Nuxt, you can create these components in the `components/` directory for them to be automatically available across your application without having to explicitly import them.
```
Comment (danielroe) [inline docs/content/1.getting-started/3.views.md:27]:
```suggestion
Most components are reusable pieces of the user interface, like buttons and menus. In Nuxt, you can create these components in the `components/` directory, and they will be automatically available across your application without having to explicitly import them.
```

File: docs/content/1.getting-started/3.views.md [hunk 4]
```diff
@@ -1,13 +1,130 @@
...
+</template>
+```
+
+::
+
+## Pages
+
+![Pages are views tied to a specific route](/img/getting-started/views/pages.svg)
+
+Pages represent views use for each specific route pattern. Every file in the `pages/` directory represents a different route displaying it's content.
```
Comment (danielroe) [inline docs/content/1.getting-started/3.views.md:56]:
```suggestion
Pages represent views use for each specific route pattern. Every file in the `pages/` directory represents a different route displaying its content.
```

File: docs/content/1.getting-started/3.views.md [hunk 5]
```diff
@@ -1,13 +1,130 @@
...
+::alert
+You will learn more about pages in the [Routing section](/getting-started/routing)
 ::
 
-::ReadMore{link="/guide/directory-structure/pages"}
+## Layouts
+
+![Layouts are wrapper around pages](/img/getting-started/views/layouts.svg)
+
+Layouts are wrapper around pages that contain a common User Interface for several pages, such as a header and footer display. Layouts are Vue files using `<slot />` components to display the **pages** content. The `layout/default.vue` file will be used by default. Custom layouts can be set as part of your page metadata.
```
Comment (danielroe) [inline docs/content/1.getting-started/3.views.md:91]:
```suggestion
Layouts are wrappers around pages that contain a common User Interface for several pages, such as a header and footer display. Layouts are Vue files using `<slot />` components to display the **page** content. The `layout/default.vue` file will be used by default. Custom layouts can be set as part of your page metadata.
```

**Review by danielroe — APPROVED** [review]
This is great! Just a few tiny tweaks.

**Pull request merged**